src/platform/backends/qemu/dnsmasq_server.cpp

```diff
@@ -47,12 +56,11 @@ auto start_dnsmasq_process(const QDir& data_dir, const QString& bridge_name, con
 }
 } // namespace
 
-mp::DNSMasqServer::DNSMasqServer(const Path& path, const QString& bridge_name, const IPAddress& bridge_addr,
+mp::DNSMasqServer::DNSMasqServer(const Path& path, const std::string& bridge_name, const IPAddress& bridge_addr,
```

Comment:
Again, not sure why `bridge_name` is now a `std::string`.

Reply:
Just the usual class boundary.

Reply:
lol, well, in this case, I prefer to leave them all as QStrings since everything consumes them as QStrings :)